packages/components/psammead-navigation/src/index.jsx

```diff
@@ -197,8 +183,27 @@ export const NavigationLi = ({
   </StyledListItem>
 );
 
-const Navigation = ({ children, script, skipLinkText, service }) => (
-  <StyledNav role="navigation">
+const StyledNav = styled.nav`
+  background-color: ${C_POSTBOX};
+
+  ${StyledListItem} {
```

Comment:
This could be simplified to 
```jsx
${({ dir }) => css`
            &::after {
              ${dir === 'ltr' ? 'left' : 'right'}: 0;
            }`
```

Reply:
Done 🚀